sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cloudfoundry/environment/Constants.java

```diff
@@ -6,47 +6,52 @@
 /**
  * utils class of Constants
  */
-public class Constants {
+final class Constants {
+
+    private Constants() {
+
+    }
+
     // Property namespaces
     /**
      * ServiceBus namespace
      */
-    public static final String NAMESPACE_SERVICE_BUS = "azure.servicebus";
+    static final String NAMESPACE_SERVICE_BUS = "azure.servicebus";
 
     /**
      * Document DB namespace
```

Comment:
also change the doc?

Reply:
after a second thought, I will change the name back to DocumentDB for this namespace has been there for a while, and if we are going to support Cosmos in cloud foundry, it's better for us to add a new namespace and also keep the old one.